config/config.py

```diff
@@ -11,3 +11,43 @@
     "https://api.regional-table.region-services.aws.a2z.com/index.json"
 )
 aws_services_json_file = "providers/aws/aws_regions_services.json"
+
+
+csv_header = [
+    "PROVIDER",
+    "PROFILE",
+    "ACCOUNT_ID",
+    "REGION",
+    "CHECKID",
+    "CHECKNAME",
+    "CHECKTITLE",
+    "STATUS",
+    "RESULT_EXTENDED",
```

Comment:
Why some fields has an `_` and other fields don't?

We need to set the same pattern for every header, what do you think?

Reply:
Need to match it with latest docs